public/plugins/octokat/LICENSE.md

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2015 Philip Schatz
```

Comment:
I originally had an issue pulling this library through npm - however here is the workaround: https://github.com/webpack/webpack-dev-server/issues/66.